docs/content/development/datasketches-aggregators.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: doc_page
+---
+
+## DataSketches aggregator
+Druid aggregators based on [datasketches]()http://datasketches.github.io/) library. Note that sketch algorithms are approxiate, see details in the [datasketches doc](http://datasketches.github.io/docs/theChallenge.html).
+
+### Aggregators
+
+```json
+{
+  "type" : "thetaSketch",
+  "name" : <output_name>,
+  "fieldName" : <metric_name>,
+
+  //following field is optional, default = false. This should only be used at
+  //indexing time if your input data contains theta sketch objects.
+  //that would be the case if you use datasketches library outside of Druid,
+  //say with Pig/Hive, to produce the data that you are ingesting into Druid
+  "isInputThetaSketch": true/false
+
+  //following field is optional, default = 16384. must be a power of 2.
+  //At query time, size value must be greater than or equal to the value
+  //used at indexing time. Internally, size refers to the maximum number
+  //of entries sketch object will retain, higher size would mean higher
+  //accuracy but higher space needed to store those sketches.
+  //See [theta-size](http://datasketches.github.io/docs/ThetaSize.html) for details.
+  //In general, I would recommend just sticking to default size which has worked well.
+  "size": 16384
+ }
+```
+
+### Post Aggregators
+
+#### Sketch Estimator
+```json
+{ "type"  : "thetaSketchEstimate", "name": <output name>, "fieldName"  : <the name field value of the sketchMerge aggregator>}
+```
+
+#### Sketch Operations
+```json
+{ "type"  : "thetaSketchSetOp", "name": <output name>, "func": <UNION|INTERSECT|NOT>, "fields"  : <the name field value of the thetaSketch aggregators>}
+```
+
```

Comment:
can we provide a concrete example of how to do retention analysis with theta sketches? With a Druid query